<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findDialog</class>
 <widget class="QDialog" name="findDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>findDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查找内容</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="casecheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>忽略大小写</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="backcheckBox">
     <property name="text">
      <string>从后向前查找</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>findDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findButton</sender>
   <signal>clicked()</signal>
   <receiver>findDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
